jmclient/test/test_valid_addresses.py

```diff
@@ -84,6 +86,71 @@ def test_invalid_bech32_addresses():
         res = address_valid_somewhere(iva)
         assert res == False, "Incorrectly validated address: " + iva
 
+def test_valid_bip341_scriptpubkeys_addresses():
+    btc.select_chain_params("bitcoin")
```

Comment:
Note that `select_chain_params` is global, that is, it might affect any code executing in parallel. More clean way would be to use `with ChainParams('bitcoin'):`

Reply:
This might not make difference for this particular code, but I wanted to mention it simply to tell that `ChainParams` exist

Reply:
> Note that `select_chain_params` is global, that is, it might affect any code executing in parallel.

Sure, I'm aware; re: in parallel we don't do that in tests (yet?) but good point.

> `with ChainParams('bitcoin'):`

however I was *not* aware of that at all, which is a very helpful point! I will probably make a PR using that in tests later (there are a few tests where we switch testnet/mainnet, as you see). Thanks.

Reply:
Oh I see it's in the README. must have missed that.

Reply:
made use of this tip, thanks!